### PR TITLE
Restrict admin to VPN users

### PR DESCRIPTION
**Pull request by vertism**

### CODE REVIEW

File: public/403.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  <body>
+    <div class="container">
+      <div class="content">
+        <div class="title">You are not authorised to access this application</div>
+        <div>
+          <br>
+          <br>
+          <b>
+            <a href="mailto:dex-engage@ministryofjustice.zendesk.com?subject=Access%20support%20request%20Find%20Unclaimed%20Money" target="_top">Contact Us</a>
```
Comment (vertism) [inline public/403.html:15]: This email might need to be changed but it matches what is on the current app.

File: public/403.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  </head>
```
Comment (digitalali-moj) [inline public/403.html:5]: Can you please add Google Tag Manager here? 



--
  | <script>(function(w,d,s,l,i){w[l]=w[l]\|\|[];w[l].push({'gtm.start':
  | new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
  | j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
  | 'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
  | })(window,document,'script','dataLayer','GTM-N9PZZHW');</script>
  |

File: public/403.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  <body>
+    <div class="container">
```
Comment (digitalali-moj) [inline public/403.html:8]: Please add the noscript bit here for Google Tag Manager (GTM for all pages)



--
  | <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-N9PZZHW"
  | height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
  |

**Review by digitalali-moj — COMMENTED** [review]
Hiya - might be easier just to get GTM in now, but happy for it to go in later if it makes sense too.

F/e looks good to me.

**Comment by vertism** [comment]
> Hiya - might be easier just to get GTM in now, but happy for it to go in later if it makes sense too.
> 
> F/e looks good to me.

I have a note that GTM needs adding, thanks,

**Pull request merged**